client/src/components/faqBlock/faqconfig.json

```diff
@@ -10,11 +10,11 @@
         "Who is the intended audience for this event?": "This fair is meant for students interested in the tech field and companies hiring for tech roles like Software Developer, Data Scientist, ML Engineer and more - not restricted to Engineering or CS students or 'purely' tech companies."
     },
     "Companies/Orgs": {
-        "How do I register and by when?": "You can register by reaching out to Umer at external@compeclub.com. The deadline is December 15th but the early pricing deadline is October 20th.",
+        "How do I register and by when?": "You can register by reaching out to Umer at external@compeclub.com. The deadline is July 20th but the early pricing deadline is September 30th.",
         "What is the Cancellation Policy?": "Grace Period: full refund will be granted if written notification of cancellation is received via email within 48-hours of registration. Cancellations received after 5:00 PM (MT) on December 15th 2023 are ineligible for grace period. Cancellations made outside of the 48-hour grace period but before 5:00 PM (MST) on December 16th, 2022 will be charged a $100 cancellation fee to cover administration and processing costs. Cancellations made after 5:00 PM (MT) on December 15th, 2023 will be charged the full fee.",
-        "How early should I show up to set up my booth for the in-person fair?": "Please plan to be there by 10:30 AM at the latest so that we can open the doors to students at 11 AM. You can also ship any marketing/promotional material in advance. Please let us know by December 15th if you need this.",
+        "How early should I show up to set up my booth for the in-person fair?": "Please plan to be there by 11:30 AM at the latest so that we can open the doors to students at 12 PM. You can also ship any marketing/promotional material in advance. Please let us know by December 15th if you need this.",
```

Comment:
We still need to change the date here as well... not sure what a good choice would be but I think September 30th is fine